.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+
+  - repo: local
+    hooks:
+      - id: nix-fmt
+        name: Format nix files
+        language: system
+        entry: nix fmt
+
+  - repo: https://github.com/hukkin/mdformat
+    rev: 0.7.22
+    hooks:
+      - id: mdformat
+        additional_dependencies:
+          - mdformat-gfm
+          - mdformat-toc
+          - mdformat-footnote
```

Comment:
Is everyone fine with these plugins ? See [CONTRIBUTING.md] for how it would look like.


[CONTRIBUTING.md]: https://github.com/snugnug/hjem-rum/pull/33/files#diff-4c6b93aa75d5affde60dc3849606c9acd75ed444d52e99f3055fc0c7aa77e9e0

Reply:
I'm okay with it, but I'm not sure if we want to add the TOC for every markdown files. The ones that will be in `content` (sample directory name, still subject to future changes) from #31 might not want one, but I'd be okay with the rest :) if you can specify only adding a TOC for some markdown files that would be great.